src/lib/components/ui/InstallBanner.svelte

```diff
@@ -42,7 +42,11 @@
         return localStorage.getItem("install-banner-dismissed") === "true"
     }
 
-    let showBanner = !(isElectron() || isTauri() || isCapacitor() || isBannerClosed())
+    function isMobile(): boolean {
+        return /Android|iPhone|iPad|BlackBerry|IEMobile|Opera Mini/i.test(navigator.userAgent)
```

Comment:
well i mean if we add IEmobile maybe we need to add all of these

```suggestion
        return /Android|iPhone|iPad|BlackBerry|IEMobile|Opera Mini|Windows Phone|webOS|Silk|SamsungBrowser|KaiOS/i.test(navigator.userAgent)
```

or instead of that we could filter by screen size instead of listing all the user strings

```isMobile: boolean = window.matchMedia("only screen and (max-width: X px)").matches```